hack/ocm-minikube.sh

```diff
@@ -317,10 +317,14 @@ ocm_subscription_operator_deploy_spoke_nonhub()
 	ocm_subscription_operator_deploy_spoke $1
 	kubectl config use-context ${1}
 	ocm_subscription_operator_checkout
+	mkdir -p multicloud-operators-subscription/munge-manifests
+	cp multicloud-operators-subscription/deploy/managed/operator.yaml multicloud-operators-subscription/munge-manifests/operator.yaml
 	MANAGED_CLUSTER_NAME=${1}\
 	HUB_KUBECONFIG=${HUB_KUBECONFIG}\
 	USE_VENDORIZED_BUILD_HARNESS=faked\
 	make -C multicloud-operators-subscription deploy-community-managed
+	cp multicloud-operators-subscription/munge-manifests/operator.yaml multicloud-operators-subscription/deploy/managed/operator.yaml
```

Comment:
(nit) If there were an interruption, e.g. ctrl+c during the make, the original file may not be restored.  This is fine if the environment is not used again, but if it is, one way to handle some interruptions is with signal traps.  A stack of commands to perform prior program termination, `exit_stack`, may be used for this purpose.